ext/ckeditor4/CRM/Ckeditor4/Upgrader.php

```diff
@@ -0,0 +1,141 @@
+<?php
+use CRM_Ckeditor4_ExtensionUtil as E;
+
+/**
+ * Collection of upgrade steps.
+ */
+class CRM_Ckeditor4_Upgrader extends CRM_Ckeditor4_Upgrader_Base {
+
+  // By convention, functions that look like "function upgrade_NNNN()" are
+  // upgrade tasks. They are executed in order (like Drupal's hook_update_N).
+
+  /**
+   * Example: Run an external SQL script when the module is installed.
+   */
+  public function install() {
+    CRM_Core_BAO_OptionValue::ensureOptionValueExists([
+      'option_group_id' => 'wysiwyg_editor',
+      'label' => 'CKEditor',
+      'name' => 'CKEditor',
+      'is_default' => 1,
+    ]);
+  }
+
+  /**
+   * Example: Work with entities usually not available during the install step.
+   *
+   * This method can be used for any post-install tasks. For example, if a step
+   * of your installation depends on accessing an entity that is itself
+   * created during the installation (e.g., a setting or a managed entity), do
+   * so here to avoid order of operation problems.
+   */
+  // public function postInstall() {
+  //  $customFieldId = civicrm_api3('CustomField', 'getvalue', array(
+  //    'return' => array("id"),
+  //    'name' => "customFieldCreatedViaManagedHook",
+  //  ));
+  //  civicrm_api3('Setting', 'create', array(
+  //    'myWeirdFieldSetting' => array('id' => $customFieldId, 'weirdness' => 1),
+  //  ));
+  // }
+
+  /**
+   * Example: Run an external SQL script when the module is uninstalled.
+   */
+  public function uninstall() {
+    civicrm_api3('Setting', 'create', ['editor_id' => 'Textarea', 'domain_id' => 'all']);
```

Comment:
I think this should be done conditionally. You only need to clobber the setting if it's currently set to 'CKEditor'.